### ISSUE TITLE
Not working with a partial file

### ISSUE THREAD

**Issue by tim-toga** [aki77/vscode-rails-i18n#19]
Hello, thanks for this useful extension!

When an underscore is added to the beginning of the view file name, it is not recognized correctly. Rails partial files have an underscore at the beginning.

Thanks

**Comment by aki77** [comment]
Thanks for the feedback.
It would be easier for us to answer if you could provide us with details about what is occurring, with a screenshot or screencast.

**Comment by tim-toga** [comment]
```ruby
# app/views/home/_info.html.erb

<%= t(".title") %>
```

```yaml
# config/locales/en.yml

en:
  home:
    info:
      title: Title
```
Goto definition or Hover doesn't work in this case.

**Comment by LukaszPiatkowski27** [comment]
my solution: https://github.com/LukaszPiatkowski27/vscode-rails-i18n

**Comment by aki77** [comment]
Fixed in v0.8.0.
